src/e2e-test/resources/pluginParameters.properties

```diff
@@ -1,6 +1,6 @@
 projectId=cdf-athena
 datasetprojectId=cdf-athena
-dataset=bq_automation
+dataset=ankitdataset
```

Comment:
Change the name of the dataset, don't create a dataset with your name.

Reply:
done.